src/multiselect.js

```diff
@@ -41,6 +41,8 @@ export class Multiselect {
     this.useCopyPasteMenu_ = true;
     this.multiFieldUpdate_ = true;
     this.multiSelectKeys_ = ['shift'];
+    this.keyboardNavKeys_ = ['ctrl', 'shift', 'k'];
```

Comment:
Is this configurable at the keyboard navigation plugin side? In any case, it would be good to make this configurable just in case.

Reply:
It is configurable on the keyboard navigation plugin side. How do we make it configurable? Do we just write a comment saying that this should be configured based on what the keyboard navigation is set to?

Reply:
Just like multiSelectKeys, make it configurable via options.